pkg/platform/platform.go

```diff
@@ -61,3 +62,18 @@ func GetPlatform(name string) (Platform, error) {
 	}
 	return platform, nil
 }
+
+// AppendTags append lokoctl-version tag to given tags map.
+func AppendTags(tags *map[string]string) {
```

Comment:
Now that we're at it, can we change it to a better name?

For example: `AppendVersionTag` or something similar?

Reply:
Done.